test/unit_decompose.jl

```diff
@@ -9,7 +9,7 @@ Y = [1.92664 1.00674 1.0731 -0.995149;
     -2.43723 0.782825 -3.99255 3.93324]
 Y = Y * CPA([BallInf([0.767292, 0.936613], 0.1), BallInf([0.734104, 0.87296], 0.1)])
 Ω0 = CH(X, Y)
-dec = Approximations.decompose(Ω0)
+dec = Approximations.decompose(Ω0, HPolygon)
```

Comment:
can you make the second argument optional?

Reply:
Okay, same as below.